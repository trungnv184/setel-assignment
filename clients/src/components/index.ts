export { default as Header } from './Header/Header';
export { default as ProductItem } from './ProductItem/ProductItem';
export { default as ProductList } from './ProductList/ProductList';
export { default as Layout } from './Layout/Layout';
export { default as CartList } from './Cart/CartList';
export { default as MenuBar } from './MenuBar/MenuBar';
export { default as ErrorMessage } from './ErrorMessage/ErrorMessage';
export { default as OrderForm } from './OrderForm/OrderForm';
export { StyledAppContainer } from './Common/Style';
