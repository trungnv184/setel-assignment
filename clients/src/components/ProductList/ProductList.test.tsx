import { render } from '@testing-library/react';
import { ProductList } from 'components';
import { MockedProvider } from '@apollo/client/testing';
import { PRODUCTS_MOCK } from 'test/mocks/products.mock';

describe('ProductList', () => {
  test('should match snapshot', () => {
    const { container } = render(
      <MockedProvider mocks={[]}>
        <ProductList products={PRODUCTS_MOCK} />;
      </MockedProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
