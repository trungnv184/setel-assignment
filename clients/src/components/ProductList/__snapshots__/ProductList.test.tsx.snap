// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductList should match snapshot 1`] = `<div />`;
